403 error with module 3 : handling

Hi there !

I am having trouble with the third module.
It runs without any error when I type `node program.js`, when I try `makemehapi run progam.js`, it outputs the html. But when I do `makemehapi verify program.js` it print "Status code 403 returned from url http://localhost:55487, expected 200."

I'm running last Linux Mint. node v6.9.1, npm 3.10.8.

I hope we can work it out :)
 
Thanks in advance for the help I'll have.

```sh
[~/MEGAsync/Developer/JavaScript/makemehapi]$ npm ls --depth 0                                                                                                                          *[4ecfcf6][ruby-2.3.0]
/Users/viktorfonic/MEGAsync/Developer/JavaScript/makemehapi
├── babel-eslint@7.1.1
├── babel-loader@6.2.8
├── babel-polyfill@6.16.0
├── babel-preset-latest@6.16.0
├── babel-preset-react@6.16.0
├── del@2.2.2
├── eslint@3.11.1
├── eslint-config-airbnb@13.0.0
├── eslint-plugin-import@2.2.0
├── UNMET PEER DEPENDENCY eslint-plugin-jsx-a11y@3.0.1
├── eslint-plugin-react@6.7.1
├── gulp@3.9.1
├── gulp-babel@6.1.2
├── gulp-eslint@3.0.1
├── hapi@16.0.1
├── inert@4.0.3
├── node-pre-gyp@0.6.32 extraneous
├── vision@4.1.1 extraneous
└── webpack-stream@3.2.0

npm ERR! peer dep missing: eslint-plugin-jsx-a11y@^2.2.3, required by eslint-config-airbnb@13.0.0
npm ERR! extraneous: node-pre-gyp@0.6.32 /Users/viktorfonic/MEGAsync/Developer/JavaScript/makemehapi/node_modules/node-pre-gyp
npm ERR! extraneous: vision@4.1.1 /Users/viktorfonic/MEGAsync/Developer/JavaScript/makemehapi/node_modules/vision
```

Note: I'm currently not using eslint package.

--- a/exercises/handling/solution/solution.js
+++ b/exercises/handling/solution/solution.js
@@ -2,7 +2,16 @@ var Hapi = require('hapi');
 var Inert = require('inert');
 var Path = require('path');
 
-var server = new Hapi.Server();
+var server = new Hapi.Server({
+    connections: {
+        routes: {
+            files: {
+                relativeTo: __dirname
+            }
+        }
+    }
+});
+
 
 server.connection({
     host: 'localhost',

